### ISSUE TITLE
"intelligens" időpontmegadás

### ISSUE THREAD

**Issue by sassbalint** [borazslo/miserend.hu#137]
@borazslo

Kiváló az oldal. :)

Egy ponton érzek hiányt: __az időpont megadásánál.__
Ez a mostani _egész nap_ legördülő, meg _"Fontos a formátum!"_ figyelmeztetés sztem nem igazán kellemes.

Azt látnám optimálisnak, ha beírhatná az ember, hogy "ma", "most", "5-kor", "egy óra múlva".
Ez lehetne hozzá a felhasználható eszköz: https://github.com/szegedai/hun-date-parser

Csak meg kéne csinálnom. Vagy más lelkes embernek. :)

**Comment by borazslo** [comment]
Ja, ez még az éles honlapon nem látszik. Egyben lesz egy naaaagy frissítése a napokban.
